bundles/org.openhab.binding.bmwconnecteddrive/README.md

```diff
@@ -0,0 +1,862 @@
+While the Air-line Distance is ~145 Kilometer the Route Distance is ~192 Kilometer.
+So Range value is the normal remaining range.
+See the Section [OpenStreetMap](#openstreetMap-widget) how the Range Radius is used to indicate the reachable Range on Map.
+Please note this is just an indicator and the effective range, especially for Electric Vehicles, 
+depends on many factors like driving style and electric consumers. 
+ 
+### Last Status Update Timestamp
+
+A timestamp is showing the last Vehicle Status update. If this isn't shown correctly please check the date settings.
+In case of Raspberry Pi execute *raspi-config*, select *Localization Options*, the *Change Time Zone*
+Select your *Geographical Area* and afterwards the correct City.
+One restart of openHAB service with *systemctl restart openhab2* is necessary in order to see the corrected Time Settings.
+ 
+Correct TimeZone is crucial for handling all Time information in openHAB and it's discussed many times in the Forum.
+See [similar discussion in the openHAB Forum](https://community.openhab.org/t/solved-wrong-local-time-how-to-change/90938) which deals with the same problem.
+
+### TroubleShooting
+
+BMW has a high range of Vehicles supported by BMWs ConnectedDrive.
+In case of any issues with this Binding please help to resolve it! 
+Please perform the following Steps:
+
+* Can you login [into ConnectedDrive](https://www.bmw-connecteddrive.com/country-region-select/country-region-selection.html) with your Credentials? _Please note this isn't the BMW Customer Portal - it's the ConnectedDrive Portal_
+* Is the Vehicle listed in your Account? _There's a one-to-one dependency from User to Vehicle_
+
+If the access to the Portal and listing of the Vehicle is checked some debug data is needed in order to identify the issue. 
+The [Example Sitemap](#Sitemap) contains the necessary items to generate Debug Data.
+
+#### My Vehicle isn't found automatically!
+
+The Vehicle isn't recognized or a predefined Vehicle doesn't show the right properties?
+For this the [Discovery Fingerprint](#bridge-channels) is needed.
+Short press on the Switch will force some logging entries which can be [normally found in the logs here](http://openhab:9001/) 
+
+#### My Vehicle shows wrong data or data is missing!
+
+Now the [Vehicle Fingerprint](#troubleshooting) is needed.
+Proceed like before, shortly press the Switch and the log contains the data.
+
+Personal Data is eliminated from the log entries so it should be possible to share them in public.
+Data like
+
+* Dealer Properties
+* Vehicle Identification Number (VIN)
+* Location Latitude / Longitude 
+
+are anonymized.
+
+After the corresponding Fingerprint is generated please [follow the instructions to raise an Issue](https://community.openhab.org/t/how-to-file-an-issue/68464) and attach the Fingerprint data!
+Your feedback is highly appreciated!
+
+Here an example of Debug Fingerprint Data from a BEV_REX Vehicle
```

Comment:
I think the example log doesn't provide any additional benefit. To keep the readme concise, I'd remove it.

Reply:
Example removed